<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>180</width>
    <height>300</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>180</width>
    <height>300</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>180</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Fan Control</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/res/fan.png</normaloff>:/res/fan.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">#centralWidget {
    background-color: #fff;
    background-image: url(:/res/bar.png);
    background-repeat: repeat-x;
    background-position: center bottom;
}

#menuButton, #temperatureButton {
    border: 0;
    background-repeat: no-repeat;
    background-position: center center;
}

#menuButton {
    background-image: url(:/res/cog.png);
    width: 18px;
    height: 18px;
}

#menuButton:pressed {
    background-image: url(:/res/cog-press.png);
}

#temperatureButton {
    background-image: url(:/res/thermometer.png);
    background-repeat: no-repeat;
    width: 18px;
    height: 24px;
}

#temperatureButton:pressed {
    background-image: url(:/res/thermometer-press.png);
}
</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>5</number>
    </property>
    <property name="topMargin">
     <number>6</number>
    </property>
    <property name="rightMargin">
     <number>5</number>
    </property>
    <property name="bottomMargin">
     <number>6</number>
    </property>
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="autoControls">
       <layout class="QGridLayout" name="gridLayout">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <property name="spacing">
         <number>0</number>
        </property>
        <item row="2" column="5">
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>3</width>
            <height>1</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="6" rowspan="4">
         <widget class="FanSpeedWidget" name="autoRightIndicator" native="true"/>
        </item>
        <item row="0" column="0" rowspan="4">
         <widget class="FanSpeedWidget" name="autoLeftIndicator" native="true"/>
        </item>
        <item row="0" column="4">
         <widget class="QLabel" name="autoRightLabel">
          <property name="font">
           <font>
            <family>Segoe UI</family>
            <pointsize>9</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Right</string>
          </property>
          <property name="textFormat">
           <enum>Qt::PlainText</enum>
          </property>
         </widget>
        </item>
        <item row="2" column="4">
         <widget class="QxtSpanSlider" name="autoRightSlider">
          <property name="minimum">
           <number>1200</number>
          </property>
          <property name="maximum">
           <number>6000</number>
          </property>
          <property name="singleStep">
           <number>100</number>
          </property>
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="tickPosition">
           <enum>QSlider::TicksBelow</enum>
          </property>
          <property name="tickInterval">
           <number>500</number>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QLabel" name="autoLeftLabel">
          <property name="font">
           <font>
            <family>Segoe UI</family>
            <pointsize>9</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Left</string>
          </property>
          <property name="textFormat">
           <enum>Qt::PlainText</enum>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="2" column="3">
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>8</width>
            <height>1</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="1" column="3">
         <spacer name="verticalSpacer_5">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>1</width>
            <height>6</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="2" column="2">
         <widget class="QxtSpanSlider" name="autoLeftSlider">
          <property name="minimum">
           <number>1200</number>
          </property>
          <property name="maximum">
           <number>6000</number>
          </property>
          <property name="singleStep">
           <number>100</number>
          </property>
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="tickPosition">
           <enum>QSlider::TicksAbove</enum>
          </property>
          <property name="tickInterval">
           <number>500</number>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>3</width>
            <height>1</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="3" column="3">
         <spacer name="verticalSpacer_3">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>1</width>
            <height>5</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="manualControls">
       <layout class="QGridLayout" name="gridLayout_2">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <property name="spacing">
         <number>0</number>
        </property>
        <item row="3" column="2">
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>1</width>
            <height>8</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="4" rowspan="4">
         <widget class="FanSpeedWidget" name="manualRightIndicator" native="true"/>
        </item>
        <item row="0" column="0" rowspan="4">
         <widget class="FanSpeedWidget" name="manualLeftIndicator" native="true"/>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="manualLeftLabel">
          <property name="font">
           <font>
            <family>Segoe UI</family>
            <pointsize>9</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Left</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QLabel" name="manualRightLabel">
          <property name="font">
           <font>
            <family>Segoe UI</family>
            <pointsize>9</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Right</string>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>8</width>
            <height>0</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="2" column="1">
         <widget class="QSlider" name="manualLeftSlider">
          <property name="minimum">
           <number>1200</number>
          </property>
          <property name="maximum">
           <number>6000</number>
          </property>
          <property name="singleStep">
           <number>100</number>
          </property>
          <property name="value">
           <number>3000</number>
          </property>
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="tickPosition">
           <enum>QSlider::TicksAbove</enum>
          </property>
          <property name="tickInterval">
           <number>500</number>
          </property>
         </widget>
        </item>
        <item row="2" column="3">
         <widget class="QSlider" name="manualRightSlider">
          <property name="minimum">
           <number>1200</number>
          </property>
          <property name="maximum">
           <number>6000</number>
          </property>
          <property name="singleStep">
           <number>100</number>
          </property>
          <property name="value">
           <number>3000</number>
          </property>
          <property name="tracking">
           <bool>true</bool>
          </property>
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="tickPosition">
           <enum>QSlider::TicksBelow</enum>
          </property>
          <property name="tickInterval">
           <number>500</number>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <spacer name="verticalSpacer_4">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>1</width>
            <height>6</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>14</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QHBoxLayout" name="barLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>4</number>
      </property>
      <property name="rightMargin">
       <number>4</number>
      </property>
      <property name="bottomMargin">
       <number>2</number>
      </property>
      <item>
       <widget class="QPushButton" name="temperatureButton">
        <property name="minimumSize">
         <size>
          <width>18</width>
          <height>24</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>18</width>
          <height>24</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="iconSize">
         <size>
          <width>10</width>
          <height>24</height>
         </size>
        </property>
        <property name="autoRepeatDelay">
         <number>313</number>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="tempSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Expanding</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>1</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="tempLabel">
        <property name="font">
         <font>
          <family>Segoe UI</family>
          <pointsize>9</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Loading...</string>
        </property>
        <property name="textFormat">
         <enum>Qt::RichText</enum>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="toolbarSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Expanding</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>1</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="menuButton">
        <property name="minimumSize">
         <size>
          <width>18</width>
          <height>18</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>18</width>
          <height>18</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="iconSize">
         <size>
          <width>18</width>
          <height>18</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QxtSpanSlider</class>
   <extends>QSlider</extends>
   <header>qxtspanslider.h</header>
   <slots>
    <signal>spanChanged(int,int)</signal>
   </slots>
  </customwidget>
  <customwidget>
   <class>FanSpeedWidget</class>
   <extends>QWidget</extends>
   <header>fanspeedwidget.h</header>
   <container>1</container>
   <slots>
    <slot>setValueRange(int,int)</slot>
    <slot>setValue(int)</slot>
   </slots>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>autoLeftSlider</tabstop>
  <tabstop>autoRightSlider</tabstop>
  <tabstop>manualLeftSlider</tabstop>
  <tabstop>manualRightSlider</tabstop>
  <tabstop>temperatureButton</tabstop>
  <tabstop>menuButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>autoRightSlider</sender>
   <signal>spanChanged(int,int)</signal>
   <receiver>autoRightIndicator</receiver>
   <slot>setValueRange(int,int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>114</x>
     <y>180</y>
    </hint>
    <hint type="destinationlabel">
     <x>159</x>
     <y>180</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>autoLeftSlider</sender>
   <signal>spanChanged(int,int)</signal>
   <receiver>autoLeftIndicator</receiver>
   <slot>setValueRange(int,int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>83</x>
     <y>188</y>
    </hint>
    <hint type="destinationlabel">
     <x>40</x>
     <y>180</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>manualLeftSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>manualLeftIndicator</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>83</x>
     <y>207</y>
    </hint>
    <hint type="destinationlabel">
     <x>40</x>
     <y>180</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>manualRightSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>manualRightIndicator</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>114</x>
     <y>180</y>
    </hint>
    <hint type="destinationlabel">
     <x>159</x>
     <y>180</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
